### ISSUE TITLE
TDIC fails to launch TCL

### ISSUE THREAD

**Issue by mwinkel-dev** [MDSplus/mdsplus#2663]
**Affiliation**
MIT PSFC

**Version(s) Affected**
Alpha, but surely in all versions

**Platform**
Ubuntu 20, but surely on all platforms.

**Describe the bug**
The `tdic` utility claims to be able to pass commands to `tcl`, but that feature doesn't work.   As shown below, `tdic` can't find `tcl`.

```
$ tdic
TDI> help
TDI(C) reference (October 1999)
      help func <help function>
      help func* <lists 1st line of each help>
      use func* <lists lines in comment starting with [call:]>
      type func(*) <type/p all the functions requested>
      tcl <TCL command>
      open <shot#>
      !<shell command>
      man [TDI intrinsic]
      exit (guess)
      ; behaves as in MatLab
TDI> tcl
sh: 1: /bin/tcl: not found
TDI> 
```

**To Reproduce**
1. See the code block above.

**Expected behavior**
Should be able to run `tcl` commands from `tdic`.

**Screenshots**
n/a

**Additional context**
Was found with mdsplus installed in a non-standard directory.   However, should also arise when installed in `/usr/local/mdsplus`.

Root cause is that the `tdic` source code is using the wrong environment variable and path.

**Comment by mwinkel-dev** [comment]
The proposed fix generates this correct output.

```
$tdic
TDI> tcl
TCL> set tree junk /shot=-1
TCL> dir

\JUNK::TOP

 :NUM          :TEXT        

Total of 2 nodes.
TCL> exit
TDI> exit
$
```

**Comment by mwinkel-dev** [comment]
Closed by PR #2676.
